oleesir/authors-haven
# Users should be able to view the home page

**What does this Merge request do?**

Implements feat__updateNotifications

**Description of Tasks completed?**

Implements feat__createHomePage

**How should this be manually tested?**

- git fetch on your local machine and check out to feat__createHomePage branch
- Run "yarn start" to start the development server

**Screenshots (if appropriate)**

<img width="1440" alt="Screenshot 2021-08-22 at 8 41 36 AM" src="https://user-images.githubusercontent.com/17679511/130346714-87ef94d9-a29f-4d60-b155-6d205ca926c9.png">
